packages/core/src/index.ts

```diff
@@ -203,6 +203,8 @@ class Hopr extends EventEmitter {
         dht: KadDHT
       },
       config: {
+        // @ts-ignore
+        protocolPrefix: 'hopr',
```

Comment:
/cc @andreykiryushkin possibly relevant for the environments work

Reply:
Nope, this one isn't relevant because it only makes *all* hopr nodes incompatible with *all* other libp2p nodes, such as IPFS nodes. But we can use it for network seperation

Reply:
ok, nevertheless good to know